index.js

```diff
@@ -58,11 +58,15 @@ module.exports = class Hyperdrive extends EventEmitter {
   }
 
   findingPeers () {
-    return this.db.feed.findingPeers()
+    return this.db.corestore.findingPeers()
   }
 
-  update () {
-    return this.db.feed.update()
+  replicate (opts) {
+    return this.db.corestore.replicate(opts)
+  }
+
+  update (opts) {
+    return this.db.feed.update(opts)
```

Comment:
It was `this.db.feed.update()`, but I think it should actually do a bee update like `this.db.update()`?

Reply:
Because the bee will internally do `this.feed.update({ ifAvailable: true, hash: false })` so it should keep the behaviour of `ifAvailable`, etc